product_configurator_wizard/wizard/product_configurator.py

```diff
@@ -527,12 +582,48 @@ def add_dynamic_fields(self, res, dynamic_fields, wiz):
                 orm.setup_modifiers(node)
                 xml_dynamic_form.append(node)
 
+            node = etree.Element(
+                "field",
+                name=ro_field,
+                invisible='1',
+            )
+            orm.setup_modifiers(node)
+            xml_dynamic_form.append(node)
+
+            node = etree.Element(
+                "field",
+                name=reqd_field,
+                invisible='1',
+            )
+            orm.setup_modifiers(node)
+            xml_dynamic_form.append(node)
+
+            node = etree.Element(
+                "field",
+                name=invis_field,
+                invisible='1',
+            )
+            orm.setup_modifiers(node)
+            xml_dynamic_form.append(node)
+
         return xml_view
 
+    @api.multi
+    def get_cfg_step(self):
```

Comment:
Add a docstring, in my version I have something like:         """Attempt to return product.config.step.line object that has the id
        of the wizard state stored as string"""